content/layering/layering.adoc

```diff
@@ -8,7 +8,31 @@
 :numbered:
 :homepage https://github.com/projectatomic/container-best-practices:
 
-Guidelines on creating layers.
+This chapter provides guidelines on creating layers.
+
+== Minimizing the Number of Layers
+
+Not having too many layers does not necessarily ensure good readability as for minimizing the number of layers you will need to chain groups of commands. In order to find balance between the number of layers and easy readability, you will have to find the best strategy.
+
+In general, a new layer is created for every new instruction defined. However, this does not necessarily mean that one instruction should be associated with only one command or definition.
+To ensure transparency and good overview of the layer content, it is preferable to group related operations in one layer. For example, take this snippet from the https://github.com/openshift/sti-python/blob/master/3.3/Dockerfile[sti-python Dockerfile]:
```

Comment:
i would avoid linking to this repository as it might change under your hands and the content will get outdated... i'm OK with the snipped below. Just say that it was taken from the OpenShift Python 3.3 image.